src/pipeline/pipelineExplorer.ts

```diff
@@ -4,21 +4,22 @@
  *-----------------------------------------------------------------------------------------------*/
 
 import { TreeDataProvider, TreeView, Event, EventEmitter, TreeItem, ProviderResult, Disposable, window, extensions, commands, Uri, version } from 'vscode';
-import { Tkn, TektonNode, TknImpl, MoreNode } from '../tkn';
+import { Tkn, TektonNode, MoreNode, tknInstance } from '../tkn';
 import { WatchUtil, FileContentChangeNotifier } from '../util/watch';
 import { Platform } from '../util/platform';
 import * as path from 'path';
 
 const kubeConfigFolder: string = path.join(Platform.getUserHomePath(), '.kube');
 
 export class PipelineExplorer implements TreeDataProvider<TektonNode>, Disposable {
-  private static tkn: Tkn = TknImpl.Instance;
+  private tkn: Tkn;
   private treeView: TreeView<TektonNode>;
   private fsw: FileContentChangeNotifier;
   private onDidChangeTreeDataEmitter: EventEmitter<TektonNode | undefined> = new EventEmitter<TektonNode | undefined>();
   readonly onDidChangeTreeData: Event<TektonNode | undefined> = this.onDidChangeTreeDataEmitter.event;
 
   constructor() {
+    this.tkn = tknInstance
```

Comment:
why you assign `tknInstance` to `tkn` field? You can use `tknInstance` in any place of this module.